hack/test-go.sh

```diff
@@ -47,6 +48,7 @@ detect_races="${DETECT_RACES:-true}"
 coverage_output_dir="${COVERAGE_OUTPUT_DIR:-}"
 coverage_spec="${COVERAGE_SPEC:--cover -covermode atomic}"
 gotest_flags="${GOTEST_FLAGS:-}"
+gotestbinary_flags="${GOTESTBINARY_FLAGS:-}"
```

Comment:
why isn't this gotestflags?

Reply:
This is my only comment Bash-wise

Reply:
The current file in master has a typo `GOTESTFLAGS` vs `GOTEST_FLAGS`.  If you correct the typo, the runner breaks because you cannot pass arguments to a test binary without using `-args ...`.  So I do not understand how to pass arguments (`sub.timeout`) to test-go and somehow have the test runner honor that.

`GOTEST_FLAGS="-v -run TestIntegration/${1:-.*}"` works.

These do not work:

`"-v -run TestIntegration/${1:-.*}" -args -sub.timeout=120s"`
`"-v -run TestIntegration/${1:-.*}" -sub.timeout=120s"`

Reply:
> The command-line package list, if present, must appear before any
> flag not known to the go test command. Continuing the example above,
> the package list would have to appear before -myflag, but could appear
> on either side of -v.
> —from `go test -help`

test-go.sh uses GOTEST_FLAGS before a package name, therefore `-sub.timeout=120s` cannot be used in GOTEST_FLAGS. On the other side, we can try to change the order of arguments in test-go.sh.